### ISSUE TITLE
sourcemap of import not served/deployed

### ISSUE THREAD

**Issue by declann** [observablehq/framework#832]
Hi,

Really liking playing around with Framework, congrats on an awesome release!

We might like sourcemaps to go out of fashion(?) but I generate ESM and keep a sourcemap.

I'm importing that ESM using a regular import:

`import * as model from './cul/calcs_esm/cul_scope_0.js'`

The [file](https://declann.observablehq.cloud/calculang-dx-pattern-exampling/_import/graphing-calcs/cul/calcs_esm/cul_scope_0.js?sha=eb5e2d569d78a2df19afef49bce19db00786a1fde845bdcc8ae39cdc06a720fb) ends with `//# sourceMappingURL=cul_scope_0.js.map`: a reference to the sourcemap file, but because this file isn't deployed (to `dist` or to `observablehq.cloud`) or served (with `npm run dev`), then it isn't available to devtools - see error:

![image](https://github.com/observablehq/framework/assets/353214/518bcd86-2b79-4144-982b-e4a8f0724fcb)

Should/can Framework detect and serve sourcemaps?

On Quarto I include sourcemap files as a resource and they gets served - this could be a workaround, but might need to be a more particular one given Frameworks directory structure (`_file`, `_import` etc.)

Thanks!

Declan

**Comment by Fil** [comment]
Yes we hope to add sourcemaps in the future (see e.g. #805).

**Comment by mbostock** [comment]
Yes, I think it would be reasonable to check for `//# sourceMappingURL=…` and include the referenced file in `dist` on build.
